### PR TITLE
Got rid of disposable_js

### LINKED ISSUE #539: Rename `disposable_js.py`
Would it be a good idea to clarify the name of `disposable_js.py` to reflect the function that it serves in collaboration with `modal.js`? I was thinking of something like `modal_response.py`. Does anyone else have other suggestions?

### PR DESCRIPTION
**Pull request by Seanny123**
Fixes #539

### CODE REVIEW

**Comment by tbekolay** [comment]
The commit message is a bit misleading; looks like you're just deleting `disposable_js`. Perhaps `
Split disposable_js into modal_js, static_plots`

File: nengo_gui/static_plots.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+import nengo
+from nengo.utils.ensemble import response_curves, tuning_curves
+
+class PlotInfo(object):
```
Comment (tbekolay) [inline nengo_gui/static_plots.py]: PEP8 says two blank lines before class and function definitions

**Comment by tbekolay** [comment]
Aside from the inline comment, the rearranging looks good! Note that I haven't checked it out and run it, just looked at the code.

**Comment by tcstewar** [comment]
LGTM  :)

**Pull request merged**